src/visualizations/config/adapters/dhis_highcharts/addTrendLines.js

```diff
@@ -212,114 +209,114 @@ function linear(data, decimalPlaces = 2) {
 }
 
 // Code extracted from https://github.com/Tom-Alexander/regression-js/
-function logarithmic(data) {
-    let sum = [0, 0, 0, 0],
-        results = [],
-        mean = 0
-
-    for (let n = 0, len = data.length; n < len; n++) {
-        if (data[n].x != null) {
-            data[n][0] = data[n].x
-            data[n][1] = data[n].y
-        }
-        if (data[n][1] != null) {
-            sum[0] += Math.log(data[n][0])
-            sum[1] += data[n][1] * Math.log(data[n][0])
-            sum[2] += data[n][1]
-            sum[3] += Math.pow(Math.log(data[n][0]), 2)
-        }
-    }
-
-    let B = (n * sum[1] - sum[2] * sum[0]) / (n * sum[3] - sum[0] * sum[0])
-    let A = (sum[2] - B * sum[0]) / n
-
-    for (let i = 0, len = data.length; i < len; i++) {
-        let coordinate = [data[i][0], A + B * Math.log(data[i][0])]
-
-        results.push(coordinate)
-    }
-
-    results.sort((a, b) => {
-        if (a[0] > b[0]) {
-            return 1
-        }
-        if (a[0] < b[0]) {
-            return -1
-        }
-
-        return 0
-    })
-
-    let string =
-        'y = ' +
-        Math.round(A * 100) / 100 +
-        ' + ' +
-        Math.round(B * 100) / 100 +
-        ' ln(x)'
-
-    return {
-        equation: [A, B],
-        points: results,
-        string: string,
-    }
-}
+// function logarithmic(data) {
```

Comment:
Maybe add a comment stating that these functions are currently not in use but may be needed in the future. @janhenrikoverland @edoardo Is there a good reason to hang on to this code?

Reply:
The code referenced undefined variables, so it has to be removed or commented.  I would be happy to remove it.

Reply:
Not in use so I would remove it.
@janhenrikoverland was it used in the past at some point?
This would need support in the backend for saving the correct value for the option, which does not seem to be there currently: https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/RegressionType.java.
Safe to remove then?

Reply:
Same thing for `power` below.

Reply:
@amcgee `logarithmic` and `power` are okay to remove.

Reply:
Removed